system/jlib/jutil.cpp

```diff
@@ -1887,6 +1712,44 @@ static const char *findExtension(const char *fn)
     return ret;
 }
 
+unsigned runExternalCommand(StringBuffer &output, const char *cmd, const char *input)
+{
+    try
+    {
+        Owned<IPipeProcess> pipe = createPipeProcess();
+        pipe->run(cmd, cmd, ".", input != NULL, true, true, 1024*1024);
+        if (input)
+        {
+            pipe->write(strlen(input), input);
+            pipe->closeInput();
+        }
+        char buf[1024];
+        while (true)
```

Comment:
You can. But that would be a separate bug from the one that this change is fixing.